README.md

```diff
@@ -55,24 +57,37 @@ items:
     selfLink: /apis/ptp.openshift.io/v1/namespaces/openshift-ptp/ptpoperatorconfigs/default
     uid: d7286542-34bd-4c79-8533-d01e2b25953e
   spec:
+    ptpEventConfig:
+      enableEventPublisher: true
+      transportHost: "amqp://amq-router.amq-router.svc.cluster.local"
     daemonNodeSelector:
       node-role.kubernetes.io/worker: ""
 kind: List
 metadata:
   resourceVersion: ""
   selfLink: ""
 ```
-
-
 ## PtpConfig
 
-`PtpConfig` CRD is used to define linuxptp configurations and to which node these linuxptp configurations shall be applied. The Spec of CR has two major sections. The first section `profile` contains `interface`, `ptp4lOpts` and `phc2sysOpts` options, the second `recommend` defines profile selection logic.
-
+`PtpConfig` CRD is used to define linuxptp configurations and to which node these 
+linuxptp configurations shall be applied. 
+The Spec of CR has two major sections. 
+The first section `profile` contains `interface`, `ptp4lOpts` , `phc2sysOpts`  and `ptp4lConf` options,
```

Comment:
extra space here.

Reply:
Thanks , ran the tool to check extra space and I believe everything is  fixed now